src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -257,47 +288,63 @@ private void attachCaseByCcdId(
         Long exceptionRecordId,
         String idamToken,
         String userId,
-        CaseDetails exceptionRecord
+        CaseDetails exceptionRecord,
+        boolean ignoreWarnings
     ) {
         log.info("Attaching exception record '{}' to a case by CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
 
-        CaseDetails theCase = ccdApi.getCase(targetCaseCcdRef, exceptionRecordJurisdiction);
-        List<Map<String, Object>> targetCaseDocuments = getScannedDocuments(theCase);
+        Validation<Seq<String>, ExceptionRecord> validationResult =
+            createCaseValidator.getValidation(exceptionRecord);
+        if (validationResult.isValid()
+            && validationResult.get().journeyClassification == SUPPLEMENTARY_EVIDENCE_WITH_OCR) {
+            ServiceConfigItem serviceConfigItem = getServiceConfig(exceptionRecord).get();
+            ccdCaseUpdater.updateCase(
+                validationResult.get(),
+                serviceConfigItem,
+                ignoreWarnings,
+                idamToken,
+                userId,
+                targetCaseCcdRef
+            );
+        } else {
```

Comment:
This `else` represents one of two possibilities: invalid case details or classification different than `SUPPLEMENTARY_EVIDENCE_WITH_OCR`. The 'invalid' part doesn't seem right.